README.md

```diff
@@ -0,0 +1,20 @@
+# Building
+
+## Build Pre-requisites
+
+The build instructions assume you have NimSkull, latest preferred unless
+otherwise noted, installed and on the path.
+
+## Build Instructions
+
+This command will get everything built
+(N.B.: Windows users replace `nim` with `nim.exe` in the command below)
+
+```cmd
+nim c --outdir:bin -r koch.nim all
+```
+
+The above will:
+- compile `koch`, the build command
+- place that within the local `bin/` directory
+- finally, it'll build all programs in this repository
```

Comment:
```suggestion
- finally, it'll build all programs in this repository and place their
  executables in the local `bin/` directory
```
It think it makes sense to also specify where the executable are placed (by default).

Also, this isn't entirely correct at the moment, since `skully` is currently skipped (for build performance reasons), but `kock all` really should do what the name implies, so I'll make a separate PR to make it work as advertised.

Reply:
Yeah, I knew it didn't build `skully`, I left it this was because I figured we'd change that eventually (it gets faster or we add it).